docs/tools/vdb_table/data/neo4j.json

```diff
@@ -123,30 +124,30 @@
     "comment": ""
   },
   "disk_index": {
-    "support": "none",
+    "support": "full",
```

Comment:
please provide a reference to docs/example notebook for this in "source_url" as requested by @prrao87 below